config/production.json

```diff
@@ -4,7 +4,7 @@
 		"max_static_paths": 9999
 	},
 	"dev_dot": {
-		"beta_product_slugs": ["vault", "waypoint", "well-architected-framework"],
```

Comment:
Remove this from the product array as it doesn't behave like a normal product within the product context. So we'll manually allow the pages to be built.